src/routes/RouteItem/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Redirect, Route } from "react-router-dom";
+import { PATHS } from "../paths";
+import IRoute from "../types";
+
+function RouteItem({ component: Component, ...rest }: IRoute): JSX.Element {
+    const someLoginValidation = () => true;
+
+    return (
+        <Route
+            {...rest}
+            render={({ location }) =>
```

Comment:
Otro warning: 'location' is defined but never used

Reply:
Solucionadoo